### PR TITLE
Add Girls in Tech Catalyst Conference

### PR DESCRIPTION
**Pull request by linda-lai**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ Of course! We'd love you to! See the [contribution guide](https://github.com/Rea
 | [YOW! Lambda Jam](https://lambdajam.yowconference.com.au/) | NSW | 13-May-2019 | 15-May-2019 | 22-Mar-2019 | 22-Mar-2019 | Functional Programming |
 | [Voxxed Days](https://australia.voxxeddays.com/) | VIC | 13-May-2019 | 14-May-2019 | 24-Oct-2018 | 11-Feb-2019 | Various |
 | [Voxxed Days](https://australia.voxxeddays.com/) | NSW | 16-May-2019 | 17-May-2019 | 25-Oct-2018 | 11-Feb-2019 | Various |
+| [Catalyst Conference](https://catalystaustralia.girlsintech.org/) | VIC | 15/05/2017 | 16/05/2017 | | | Women in Tech |
```
Comment (aaronpowell) [inline readme.md]: Could you use the same date format as the other entries please? `15-May-2019` and `16-May-2019` (also, I assume you mean 2019, not 2017 😉).

It makes it easier to parse when they are all consistent 😊

Comment (linda-lai) [reply readme.md]: Yes, of course sorry - my bad, thanks for the pickup 😉

**Comment by mderriey** [comment]
Hey @linda-lai, thanks for your contribution 🎉

**Comment by mderriey** [comment]
Ugh, should have squashed those commits, sorry for the noise in the Git history!

**Pull request merged**